corehq/apps/hqwebapp/static/hqwebapp/js/components/inline_edit.js

```diff
@@ -118,9 +117,7 @@ hqDefine('hqwebapp/js/components/inline_edit', [
                             self.isSaving(false);
                             self.hasError(false);
                             self.serverValue = self.readOnlyValue;
-                            if (self.postSave) {
-                                self.postSave(data);
-                            }
+                            $(document).trigger("inline-edit-save", data);
```

Comment:
Would it make sense to put a comment here or maybe up with the other comment above `self.save = function () {` that explains the rationale for using `$(document).trigger(...)`?

Reply:
yes - added in 7d81f958e44